packages/E2ETest/windows/ReactUWPTestApp/ReactUWPTestApp.csproj

```diff
@@ -184,11 +184,5 @@
     <BundleEntryFile>dist/app/index.js</BundleEntryFile>
   </PropertyGroup>
   <Import Project="$(ReactNativeWindowsDir)\PropertySheets\Bundle.targets" />
```

Comment:
>t Project="$(ReactNativeWindowsDir)\Pro [](start = 8, length = 39)

This can be removed if you import the csharpapp.targets

Reply:
or I can do it in my pr...

---
In reply to: [428799747](https://github.com/microsoft/react-native-windows/pull/4965#discussion_r428799747) [](ancestors = 428799747)

Reply:
same for bundle.props and the projectreferences.

---
In reply to: [428800047](https://github.com/microsoft/react-native-windows/pull/4965#discussion_r428800047) [](ancestors = 428800047,428799747)

Reply:
as well as the playgroup.vcxproj

---
In reply to: [428800233](https://github.com/microsoft/react-native-windows/pull/4965#discussion_r428800233) [](ancestors = 428800233,428800047,428799747)

Reply:
yeah I figured you'd change this in yours but wanted to get this in unless you're getting yours in first? :)